Dockerfile

```diff
@@ -1,19 +1,16 @@
-FROM node:6-slim
+FROM node:10-slim
 
 WORKDIR /usr/src/app
 
-COPY package.json /usr/src/app
+COPY . /usr/src/app
 
 RUN apt-get update \
-    && apt-get install -y jq --no-install-recommends \
-    && npm install --production \
-    && rm -rf /var/lib/apt/lists/* \
+    && apt-get install git -y \
+    && apt install build-essential -y \
+    && apt-get install python -y \
+    && apt-get install g++ -y \
     && npm cache clear --force \
-    && rm -rf ~/.node-gyp \
-    && rm -rf /tmp/npm-*
-
-# Keep the .git directory in order to properly report version
-COPY . /usr/src/app
+    && npm install --production
 
 ENTRYPOINT ["/usr/src/app/docker-entrypoint.sh"]
```

Comment:
At this point, could be interesting to `apt-get remove` the build-only dependencies introduced like `g++` and `git` to reduce the image size.

Reply:
Great point! I'll try that out and update.

Reply:
Updated to remove those dependencies.